web/client/reducers/mapInfo.js

```diff
@@ -55,7 +57,6 @@ function receiveResponse(state, action, type) {
     return state;
 }
 const initState = {
-    enabled: true,
```

Comment:
i think this can stay, please revert

Reply:
it should be here because default value is configured through the plugin configuration